test/sagas/session_test.js

```diff
@@ -207,6 +233,31 @@ describe("session sagas", () => {
           put(actions.setupComplete(authData))
         );
       });
+
+      it("should correctly authenticate the user when using openID", () => {
+        const authData = {
+          server: "http://server.test/v1",
+          authType: "openid-google",
+          credentials: { token: "the token" },
+        };
+        const serverInfo = {
+          ...serverInfo,
+          user: { id: "auth0:token" },
+        };
```

Comment:
I would say this situation is impossible to obtain with a real server.

If you authenticate with a provider name `google`, then your userid will be `google:tartempion`

Reply:
are you sure about that? In my tests, i got `auth0:...` back, which was a surprise to me!